### PR TITLE
feat(route): 轻之国度 文章更新阅读 #11848

### LINKED ISSUE #11848: 轻之国度 轻小说
### 类型

阅读

### 网站地址

https://www.lightnovel.us/

### 网站描述

轻小说翻译分享网站

### 需要生成什么内容？

抓取轻小说，漫画等更新内容

### 额外描述

_No response_

### 这不是重复的 RSS 请求

- [X] 我已经搜索了[现有 issue](https://github.com/DIYgod/RSSHub/issues) 和 [pull requests](https://github.com/DIYgod/RSSHub/pulls)，以确保该 RSS 尚未被请求。

### PR DESCRIPTION
**Pull request by nightmare-mio**
## Involved Issue / 该 PR 相关 Issue

Close #11848

## Example for the Proposed Route(s) / 路由地址示例

```routes
/lightnovel/歡迎來到實力至上主義的教室/3cfc2dc63f3575ee42e12823188ad1b5:1709125:0/
```

## New RSS Route Checklist / 新 RSS 路由检查表
  
- [X] New Route / 新的路由
  - [X] Follows [v2 Script Standard](https://docs.rsshub.app/joinus/advanced/script-standard) / 跟随 [v2 路由规范](https://docs.rsshub.app/zh/joinus/advanced/script-standard)
- [X] Documentation / 文档说明
- [X] Full text / 全文获取
  - [ ] Use cache / 使用缓存
- [X] Anti-bot or rate limit / 反爬/频率限制
  - [X] If yes, do your code reflect this sign? / 如果有, 是否有对应的措施?
- [X] [Date and time](https://docs.rsshub.app/joinus/advanced/pub-date) / [日期和时间](https://docs.rsshub.app/zh/joinus/advanced/pub-date)
  - [X] Parsed / 可以解析
  - [X] Correct time zone / 时区正确
- [x] New package added / 添加了新的包
- [ ] `Puppeteer`

### CODE REVIEW

File: lib/v2/lightnovel/lightNovel.js [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+                });
+                const response = await page.content();
+                page.close();
+                const $ = cheerio.load(response);
+                item.description = $('#article-main-contents').html();
+                return item;
+            })
+        )
+    );
+    browser.close();
```
Comment (TonyRL) [inline lib/v2/lightnovel/lightNovel.js:93]: Unnecessary use of puppeteer.
You can still obtain the content using `got`

File: lib/v2/lightnovel/maintainer.js [hunk 2]
```diff
@@ -0,0 +1,3 @@
+module.exports = {
+    '/:keywords/:security_key': ['nightmare-mio'],
```
Comment (TonyRL) [inline lib/v2/lightnovel/maintainer.js:2]: https://github.com/DIYgod/RSSHub/blob/86c458f9d80d92495e46c04ff7a3650f53390264/lib/v2/lightnovel/router.js#L2
```suggestion
    '/:keywords/:security_key?': ['nightmare-mio'],
```

File: lib/v2/lightnovel/radar.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
+module.exports = {
+    'lightNovel.us': {
+        _name: '轻之国度',
+        '.': [
+            {
+                title: '文章更新阅读',
+                docs: 'https://docs.rsshub.app/routes/annime#wen-zhang-geng-xing-yue-du',
```
Comment (TonyRL) [inline lib/v2/lightnovel/radar.js:7]: https://github.com/DIYgod/RSSHub/blob/86c458f9d80d92495e46c04ff7a3650f53390264/website/docs/routes/anime.mdx#L837
```suggestion
                docs: 'https://docs.rsshub.app/routes/anime#qing-zhi-guo-du',
```

File: lib/v2/lightnovel/radar.js [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+                source: '/',
+                target: '/lightNovel/:keywords/:security_key',
```
Comment (TonyRL) [inline lib/v2/lightnovel/radar.js:9]: https://github.com/DIYgod/RSSHub/blob/86c458f9d80d92495e46c04ff7a3650f53390264/website/docs/routes/anime.mdx#L841
```suggestion
                target: '/lightnovel/:keywords/:security_key',
```

File: website/docs/routes/anime.mdx [hunk 5]
```diff
@@ -834,6 +834,12 @@ You can use some RSS parsing libraries (like `feedpraser` in `Python`) to receiv
 
 <Route author="junfengP" path="/manxiaosi/book/:id" example="/manxiaosi/book/90" paramsDesc={['漫画id，漫画主页的地址栏中']} radar="1" rssbud="1"/>
 
+## 轻之国度 {#qing-zhi-guo-du}
+
+### 文章更新阅读 {#wen-zhang-geng-xing-yue-du}
+
+<Route author="nightmare-mio" path="/lightnovel/:keywords/:security_key" example="/lightnovel/歡迎來到實力至上主義的教室/3cfc2dc63f3575ee42e12823188ad1b5:1709125:0" paramsDesc={['关键字，可以模糊匹配，但最好精确匹配。默认为文章类型','cookie,由于文章有防爬，所以必须携带cookie请求。route中的cookie优先级高于环境变量cookie，取token中的security_key值']} radar="1"  rssbud="1"/>
```
Comment (TonyRL) [inline website/docs/routes/anime.mdx:841]: https://github.com/DIYgod/RSSHub/blob/86c458f9d80d92495e46c04ff7a3650f53390264/lib/v2/lightnovel/router.js#L2
```suggestion
<Route author="nightmare-mio" path="/lightnovel/:keywords/:security_key?" example="/lightnovel/歡迎來到實力至上主義的教室/3cfc2dc63f3575ee42e12823188ad1b5:1709125:0" paramsDesc={['关键字，可以模糊匹配，但最好精确匹配。默认为文章类型','cookie,由于文章有防爬，所以必须携带cookie请求。route中的cookie优先级高于环境变量cookie，取token中的security_key值']} radar="1" />
```

**Review by TonyRL — COMMENTED** [review]
https://github.com/DIYgod/RSSHub/pull/13938#discussion_r1419192069

**Comment by nightmare-mio** [comment]
> [第13938章（评论）](https://github.com/DIYgod/RSSHub/pull/13938#discussion_r1419192069)
I have made the necessary modifications

**Comment by TonyRL** [comment]
> > [第13938章（评论）](https://github.com/DIYgod/RSSHub/pull/13938#discussion_r1419192069)
> > I have made the necessary modifications

https://github.com/DIYgod/RSSHub/pull/13938#discussion_r1419192069

Please **update the docs** in `/website/i18n/zh/docusaurus-plugin-content-docs/current/install/config.md` as well.

**Comment by nightmare-mio** [comment]
> > > [第13938章（评论）](https://github.com/DIYgod/RSSHub/pull/13938#discussion_r1419192069)
> > > I have made the necessary modifications
> 
> [#13938 (comment)](https://github.com/DIYgod/RSSHub/pull/13938#discussion_r1419192069)
> 
> Please **update the docs** in `/website/i18n/zh/docusaurus-plugin-content-docs/current/install/config.md` as well.

sorry，i'm first times

**Pull request merged**